index.html

```diff
@@ -1,17 +1,13 @@
-<html>
```

Comment:
@JoshuaAKASHA you've might deleted too many html tags :D

Reply:
Hey @kenshyx, are you able to put through all of these bar cf5c8c1? If not I can close and reopen with the proper code :D

Reply:
Close and reopen 👍